.github/ISSUE_TEMPLATE/default_issue.yml

```diff
@@ -0,0 +1,57 @@
+name: Default Issue
+description: File an issue for a new feature or bug
+title: "[Issue]: "
+projects: "Grants Team Agile Planning"
```

Comment:
I would remove this line since Github doesn't actually parse this file correctly